.storybook/preview.scss

```diff
@@ -0,0 +1,3 @@
+@import '../lib/styles/normalize';
+@import '../lib/styles/design-system';
+@import '../lib/styles/storybook';
```

Comment:
How about moving this file to `/.storybook`? Would that work? It's not a part of the exported lib, so the current path stinks a bit.